app/src/main/java/org/simple/clinic/onboarding/OnboardingScreenEvents.kt

```diff
@@ -0,0 +1,5 @@
+package org.simple.clinic.onboarding
+
+import org.simple.clinic.widgets.UiEvent
+
+class GetStartedClicked : UiEvent
```

Comment:
I personally feel it's unnecessary doing that since they are already scoped to a package, and we have feature-based packaging. Maybe we should have a discussion offline and see if it's needed or not, and then rename the events later on if we decide to do it.

Reply:
> since they are already scoped to a package, and we have feature-based packaging

Packaging with features doesn't help with auto-complete because the IDE doesn't prioritize suggestions based on the package.